src/dlls/dbgshim/dbgshim.cpp

```diff
@@ -387,12 +387,12 @@ class RuntimeStartupHelper
         return hr;
     }
 
-    bool NoNullHandles(HANDLE *handleArray, DWORD arrayLength)
+    bool AreAllHandlesValid(HANDLE *handleArray, DWORD arrayLength)
     {
         for (int i = 0; i < (int)arrayLength; i++)
         {
             HANDLE h = handleArray[i];
-            if (h == NULL)
+            if (h == INVALID_HANDLE_VALUE)
```

Comment:
In the case of older CoreCLR's, this will still be NULL, so should we check both?